src/Container/SesClientFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I didn't use it anywhere in the library so for consistency I'd suggest you get rid of this.